src/components/profile/UserProgress/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Typography } from '@/components/common';
+import { Variant } from '@/components/common/Typography';
```

Comment:
```suggestion
import { type Variant } from '@/components/common/Typography';
```

I think

Reply:
also I think `@/components/common` should also re-export `Variant` so you can merge the first two imports